src/data/objects/image.yaml

```diff
@@ -39,7 +39,6 @@ schema:
   is_public:
     type: Boolean
     value: false
-    filterable: true
```

Comment:
is_public is filterable per https://bits.linode.com/LinodeAPI/apinext/blob/development/apinext/maps/image.py#L53